modules/@apostrophecms/ui/ui/apos/components/AposCellLabels.vue

```diff
@@ -3,28 +3,28 @@
     class="apos-table__cell-field apos-table__cell-field--labels"
     :class="`apos-table__cell-field--${header.name}`"
   >
-    <span v-if="manuallyPublished && item.modified && !item.submitted && item.lastPublishedAt">
+    <span v-if="manuallyPublished && draft.modified && draft.submitted && draft.lastPublishedAt">
       <AposLabel
         label="Pending Updates" class="apos-table__cell-field__label"
         tooltip="There are active changes to this document."
         :modifiers="[ 'is-success', 'is-filled' ]"
       />
     </span>
-    <span v-else-if="item.submitted">
+    <span v-else-if="draft.submitted">
```

Comment:
Why do we have this if the previous one also checks for `draft.submitted`?

Reply:
Just because it is submitted doesn't mean it's all those other things in that "and" statement. If it isn't, we have a fallback that applies to the case where it is merely submitted.